fixed a bug when validating pack-ignore files (#3528)
demisto/demisto-sdk: CHANGELOG.md

* Added support for yml hidden parameters for `xsoar_saas` marketplace, as part of the **prepare_content** command.
* Added support for custom documentation that will appear only in `xsoar_saas` marketplace, as part of the **prepare_content** command.
* Modified **prepare_content** command to be platform specific. For xsoar-saas and XSIAM regarding pack readme and integration description images in markdown files.

> * Fixed an issue where *validate* would fail with modified pack-ignore files, that were originally empty files.

Reviewer: ```suggestion
* Fixed an issue where *validate* would fail checking `.pack-ignore` files that are empty on `master`.
```
